admin_manual/configuration_files/big_file_upload_configuration.rst

```diff
@@ -2,21 +2,18 @@
 Uploading big files > 512MB
 ===========================
 
-The default maximum file size for uploads is 512MB. You can increase this 
-limit up to what your filesystem and operating system allows. There are certain 
-hard limits that cannot be exceeded:
+The default maximum file size for uploads, in ownCloud, is 512MB. 
+You can increase this limit up to the maximum file size which your filesystem, operating system, or other software allows, for example:
 
-* < 2GB on 32Bit OS-architecture
+* < 2GB on a 32Bit OS-architecture
 * < 2GB on Windows (32Bit and 64Bit)
 * < 2GB with Server Version 4.5 or older
 * < 2GB with IE6 - IE8
 * < 4GB with IE9 - IE11
+ 
+Please consult the documentation for your filesystem.
```

Comment:
@mmattel I would assume that that means ownCloud server version 4.5. Though, given version 10.0 is almost here, I'd say it's ok to remove that line, no?